README.md

```diff
@@ -1,6 +1,7 @@
 # Core Gulp Tasks
 
 **Note:** Node 0.12 or higher is required.
+**Note:** Gulp version 3.9 for both the CLI and local version are required. More info [can be found here](https://markgoodyear.com/2015/06/using-es6-with-gulp/).
```

Comment:
Thanks for the feedback @lyzadanger! Was this comment something to act upon? I'm actually a little confused.

When I opened a new terminal tab, I get:
```
$ gulp -v
[11:15:10] CLI version 0.3.0
```

Within this repo I get:
```
$ gulp -v
[11:16:02] CLI version 3.9.0
[11:16:02] Local version 3.9.0
```

I'm unclear on the relationship. Wait...so the `gulp-cli` (which is a utility for `gulp`) is at `v0.3.0`...so...shouldn't within the repo the `CLI version` also read `0.3.0`? I'm confused. LOL